CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## 35.3.1
 
+* Fix GA4 index_total parameter in various places ([PR #3358](https://github.com/alphagov/govuk_publishing_components/pull/3358))
```

Comment:
Also sorry, just realised, should this be under an `Unreleased` heading?

Reply:
Thanks, well noticed. Probably a result of the last rebase, should be fixed now.